extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialogModel.ts

```diff
@@ -131,4 +137,18 @@ export class MigrationCutoverDialogModel {
 		});
 		return files;
 	}
+
+	private async getTelemetryProps(migration: MigrationContext) {
+		return {
+			'sessionId': migration.sessionId!,
+			'subscriptionId': migration.subscription.id,
+			'resourceGroup': getResourceGroupFromId(migration.targetManagedInstance.id),
+			'sqlServerName': migration.sourceConnectionProfile.serverName,
+			'sourceDatabaseName': migration.migrationContext.properties.sourceDatabaseName,
+			'sqlServerVersion': await getSqlServerVersion(),
```

Comment:
is this version info available anywhere else, I'm concerned a Telemetry event is becoming a blocking call

Reply:
good point. I removed the sql server version from this telemetry event after realizing it's not an essential data point